src/@usermn/sdc/index.ts

```diff
@@ -0,0 +1,8 @@
+import { createVersionedSpec } from "@fig/autocomplete-helpers";
+const versionFiles = ["0.0.0"];
+export const getVersionCommand: Fig.GetVersionCommand = async (
+  executeShellCommand
+) => {
+  return await executeShellCommand("npx @usermn/sdc --version");
```

Comment:
This ensures the version is semantic:
```suggestion
  return (await executeShellCommand("npx @usermn/sdc --version")).slice(1);
```

Reply:
👍 will commit this later when I can get into the bot account